### DISCUSSION TITLE
PHP error

### DISCUSSION THREAD

**Discussion by jpnilson** [the-djmaze/snappymail#1516] [Q&A]
My log is filled with this error.  The app seems to be working ok but would like to fix if  possible.  Any ideas?

Array to string conversion at /var/www/html/custom_apps/snappymail/app/snappymail/v/2.35.4/app/libraries/snappymail/gpg/pgp.php#879

**Comment by the-djmaze** [comment]
that's a weird notice, there should be no conversion
